strategy-contract.yaml

```diff
@@ -0,0 +1,78 @@
+openapi: 3.0.1
+info:
+  title: Strategy Module
+  description: This is the Strategy Module of AEGEE-Europe
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.0.1
+paths:
+  /actionagenda/{aaId}:
+    get:
+      tags:
+      - Action Agenda
+      summary: Display a specified Action Agenda
+      description: For valid response try integer IDs with value >= 1 and <= 10.         Other
+        values will generated exceptions
+      operationId: getActionAgendaById
+      parameters:
+      - name: aaId
+        in: path
+        description: ID of Action Agenda that needs to be fetched
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          description: Get specified Action Agenda
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ActionAgenda'
+        400:
+          description: Invalid Action Agenda ID supplied
+          content: {}
+        404:
+          description: Action Agenda not found
+          content: {}
+components:
+  schemas:
+    ActionAgenda:
+      type: array
+      items:
+        $ref: '#/components/schemas/FocusArea'
+    FocusArea:
```

Comment:
New idea, but maybe it's nice to add the body_id of the responsible WG here as well? So you can later retrieve information from the WG from core (like the email address or the members)

Reply:
Agree, but I see this as a feature for the future, as it would also require us to interact with the core and starting with this seemed an easier start. But noted for the future ;)